Header header
time t
string robot_id
bool is_ready

string type
bool status

float32 x
float32 y
float32 theta
